src/jit/CMakeLists.txt

```diff
@@ -79,6 +79,120 @@ set( JIT_SOURCES
   valuenum.cpp
 )
 
+if (WIN32)
+  set( JIT_HEADERS
+    _typeinfo.h
```

Comment:
Can't we simply include all the .h files in the jit src directory? Unlike for .cpp files it does not matter if files that are not actually needed are included in the project.

Reply:
Yes, in clrjit case we can, however, I have followed convention of including each file individually. For other directories situation is not that straightforward.